quanto/tensor/core.py

```diff
@@ -313,6 +313,10 @@ def __torch_dispatch__(cls, op, types, args, kwargs=None):
     def numpy(self):
         return self.dequantize().cpu().numpy()
 
+    def to(self, *args, **kwargs):
+        self._data = self._data.to(*args, **kwargs)
+        self._scale = self._scale.to(*args, **kwargs)
+        return self
```

Comment:
I wanted to return `super().to(*args, **kwargs)` but it was causing weird behavior with tests using `QBitsTensor` and it was calling `__torch_function__` after . To reproduce, return `super().to(*args, **kwargs)`  and run `python -m pytest -sv test/nn/test_qlinear.py::test_move_qlinear`

Reply:
Tensor subclasses are a very special beast: you should not override the base Tensor methods that way, and instead do it through the dispatch.